src/telemetry/BackgroundLogger.ts

```diff
@@ -71,8 +73,8 @@ class BackgroundLogger implements Logger {
   }
 
   async error(error: unknown, data: JsonObject): Promise<void> {
-    if (isConnectionError(error) && isContentScript()) {
-      showConnectionLost();
+    if (wasContextInvalidated() && !isBackground()) {
```

Comment:
This message also applies to the options page. I suppose the page editor already has such notification, so:

```suggestion
    if (wasContextInvalidated() && !isBackground() && !isDevToolsPage()) {
```